javascript-arrays/main.js

```diff
@@ -0,0 +1,22 @@
+var colors = ['red', 'white', 'blue'];
+
+console.log('value of colors:', colors[0]);
+console.log('value of colors:', colors[1]);
+console.log('value of colors:', colors[2]);
+
+console.log('America is ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.');
+
+colors[2] = 'green';
+console.log('Mexico is ' + colors[0] + ', ' + colors[1] + ', ' + colors[2] + '.');
+
+console.log('value of colors', colors);
+
+var students = ['Danary', 'Gabriela', 'Gennesis', 'Carolina'];
+var numberOfStudents = students.length;
+
+console.log('There are ' + numberOfStudents + ' students in the class');
+
```

Comment:
Where is your `lastIndex` variable?

Reply:
<img width="709" alt="Screen Shot 2022-07-06 at 10 06 10 AM" src="https://user-images.githubusercontent.com/97363006/177605997-ba3f42bc-b299-4a5d-b18d-0c7a12d3a346.png">